resources/hab_build.rb

```diff
@@ -13,6 +13,7 @@
 property :home_dir, String
 property :auth_token, String
 property :live_stream, [TrueClass, FalseClass], default: true
+property :cleanup, [TrueClass, FalseClass], default: false
```

Comment:
I learned recently that for these kinds of properties, the allowed values should be `[true,  false]`. Please do that for `cleanup`, and while we're in this file, `live_stream` too 💃 .

Ref: https://github.com/chef-cookbooks/habitat/pull/31#discussion_r112523204